netcdf_to_gltf_converter/netcdf/netcdf_data.py

```diff
@@ -33,20 +34,35 @@ def __init__(self, data: xr.DataArray) -> None:
         """
         self._data_array = data
         self._time_var = get_coordinate_variables(data, ("time",))[0]
+        self._x_coords_var = get_coordinate_variables(data, X_STANDARD_NAMES)[0]
+        self._y_coords_var = get_coordinate_variables(data, Y_STANDARD_NAMES)[0]
```

Comment:
should there also be a z coord?

Reply:
There is `get_data_at_time` for the z-coordinates -> which are the variable values.